### PR TITLE
[WIP] Multi metric support

### PR DESCRIPTION
**Pull request by ffelten**

### CODE REVIEW

File: static/morl_deterministic_envs.png
Comment (vwxyzjn) [inline static/morl_deterministic_envs.png:1]: Lots of duplicates in chart titles and x and y labels.

Comment (ffelten) [reply static/morl_deterministic_envs.png:1]: I'll keep working on this in the coming days :).

**Comment by ffelten** [comment]
Will reopen if something clean comes out

**Comment by vwxyzjn** [comment]
Let's just merge as is since it's already functional. We can probably refactor it later. Thanks for the PR!

**Pull request merged**